.zenodo.json

```diff
@@ -1,6 +1,6 @@
 {
   "title": "remind2: The REMIND R package (2nd generation)",
-  "version": "1.99.0",
+  "version": "1.100.0",
```

Comment:
IT is not a big deal, but for changes like this we don't add to the version at this level, the recommended approach would be to use "1.99.1" instead.

Reply:
ok, I interpreted the descriptions in the list that shows up when building a library such that this level should be increased when adding variables (like the CHP variables for electricity I added).

what would be example cases when you would increase this level?

Reply:
I agree with Robert, he increased according to the description buildLibrary provides. 3rd digit increase is for bugfixes, 2nd digit increase is for new features. @Renato-Rodrigues if you feel like this is not the right way to do it, you should bring it up with RSE and ask for an adjustment of the recommendations the library gives you.

Reply:
I mean "new features" is a very vague term - so if Renato says "simply adding extensions of existing variables "cap|elect|CHP" doesn't count as new feature - for a new feature I would have to add a new class of variables such as 'idle capacity' or so" I am totally fine with that and will do so in the future. I just wanted to understand in what case I should increase 2nd digit and in what case 3rd digit.

Reply:
Don't get me wrong, i can see why this is a 3rd digit increase, not a 2nd digit increase and would myself also choose 3rd digit increase. But due to the vague description, people are left wondering and this will keep happening. So i can see why you came to that conclusion using the description.

Reply:
It was RSE that requested me to not use the subversion (2nd digit) for smaller code changes. So if you want to change the description there you should contact them directly.
They said to me that the second digit is reserved to more complex, long and/or breaking changes. Smaller changes such as small number of variables introduced or small bug fixes should only update the third digit instead.